src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -217,31 +217,19 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
 
   /**
    * Creates a set of JES inputs for a backend call.
+   * Note that duplicates input files (same gcs path) will be (re-)localized every time they are referenced.
```

Comment:
I don't understand the comment. What exactly is going on there? (I'm sure the wise @mcovarr is correct, however)